plugins/elementspath/plugin.js

```diff
@@ -222,7 +222,7 @@
 			}
 
 			var space = getSpaceElement();
-			space.setHtml( html.join( '' ) + emptyHtml );
+			space.setHtml( CKEDITOR.tools.htmlLegacyConverted(html.join( '' ) + emptyHtmld) );
```

Comment:
is this a typo? `emptyHtml` instead of `emptyHtmld`

Reply:
Yep my bad just fixed this